api/proto/teleport/legacy/types/types.proto

```diff
@@ -1118,6 +1118,24 @@ message AppSpecV3 {
   // want the app to be accessible from any of them. If `public_addr` is explicitly set in the app spec,
   // setting this value to true will overwrite that public address in the web UI.
   bool UseAnyProxyPublicAddr = 14 [(gogoproto.jsontag) = "use_any_proxy_public_addr,omitempty"];
+  // MCP contains MCP server related configurations.
+  MCP MCP = 15 [(gogoproto.jsontag) = "mcp,omitempty"];
+}
+
+// MCP contains MCP server-related configurations.
+message MCP {
+  // Command to launch stdio-based MCP servers.
+  string command = 1;
+  // Args to execute with the command.
+  repeated string args = 2;
+  // RunAsLocalUser is the local user account under which the command will be
+  // executed. Required for stdio-based MCP servers.
+  string run_as_local_user = 3;
```

Comment:
Just to make it sure: the user can be expressed as numeric ID too?

Also, I'm having second thoughts regarding the name, specifically "local" part. I know that is meant to represent this isn't a Teleport user or something, but I wonder whether any of these would be better:
- `run_as_user`
- `run_as_os_user`
- `run_as_system_user`

Just raising it for awareness.

Reply:
changed to `run_as_host_user` to be more consistent with others in the role spec.